plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2038,178 @@ def coupling_parameter(
         )
 
     return coulomb_energy / kinetic_energy
+
+
+class CollisionFrequencies:
+    @validate_quantities(
+        v_a={"none_shall_pass": True},
+        T_a={
+            "none_shall_pass": True,
+            "can_be_negative": False,
+            "equivalencies": u.temperature_energy(),
+        },
+        n_b={"can_be_negative": False},
+        T_b={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+    )
+    @particles.particle_input
+    def __init__(
+        self,
+        test_particle: particles.Particle,
+        field_particle: particles.Particle,
+        *,
+        v_a: u.cm / u.s = None,
+        T_a: u.K = None,
+        n_b: u.cm**-3,
+        T_b: u.K,
+        Coulomb_log: u.dimensionless_unscaled,
+    ):
+        r"""
+        Compute collision frequencies between test particles (labeled 'a') and field particles (labeled 'b')
+
+        Parameters
+        ----------
+        test_particle : |Particle|
+            The test particle streaming through a background of field particles.
+
+        field_particle : |Particle|
+            The background particle being interacted with.
+
+        v_a : `~astropy.units.Quantity`, optional
+            The relative velocity between particles. If not provided, T_a must be specified and
+            thermal velocity is assumed: :math:`μ v_a^2 \sim 2 k_B T_a` where
+            :math:`μ` is the reduced mass.
+
+        T_a : `~astropy.units.Quantity`, optional
+            The temperature of the test species. Only necessary if :math:`v_a` is not provided.
+
+        n_b : `~astropy.units.Quantity`
+            The number density of the background field particles.
+
+        T_b : `~astropy.units.Quantity`
+            The temperature of the background field particles.
+
+        Coulomb_log : `~astropy.units.Quantity`
+            The value of the Coulomb logarithm evaluated for the two interacting particles.
+
+        Raises
+        ------
+        `ValueError`
+            If both :math:`v_a` and :math:`T_a` are specified, or neither
+            :math:`v_a` nor :math:`T_a` are specified.
+
+        Notes
+        -----
+        The collision frequencies between two interacting particles are given by four differential equations:
+
+            slowing down - :math:`\frac{d\textbf{v}_{α}}{dt}=-ν_{s}^{α\backslashβ}\textbf{v}_{α}`
+
+            transverse diffusion - :math:`\frac{d}{dt}\left(\overline{\textbf{v}}_{α}-\overline{\textbf{v}}_{α}\right)_{⊥}^{2}=ν_{⊥}^{α\backslashβ}v_{α}^{2}`
+
+            parallel diffusion - :math:`\frac{d}{dt}\left(\overline{\textbf{v}}_{α}-\overline{\textbf{v}}_{α}\right)_{∥}^{2}=ν_{∥}^{α\backslashβ}v_{α}^{2}`
+
+            energy loss - :math:`\frac{d}{dt}v_{α}^{2}=-ν_{ϵ}^{α\backslashβ}v_{α}^{2}`
+
```

Comment:
For completeness I would write out the formulas for the collision frequencies and the definitions of the Lorentz collision frequency, x, and phi. It is important to explain what x is, I think, to understand what is happening conceptually here. 

For a future PR you could write a nice example notebook using this function in which you plot psi as a function of x and illustrate where the limiting approximations work! I made a plot like this in my thesis: 

![image](https://user-images.githubusercontent.com/32618747/186211525-d8c073cd-7b94-4f2f-bce2-e19e9c0c5552.png)